### PR TITLE
Filter out Draft pull requests

### PR DESCRIPTION
**Pull request by joelbourbon**
Added changelog.md in order to track what is introduced in every version of the extension
Whitespace cleanup
Version bump

Relates to #30

### CODE REVIEW

**Comment by joelbourbon** [comment]
I would like to test out those changes on my Azure Devops before this gets merged. But I require the owner of my team to collaborate.

Will ping back once tested!

File: src/vss-pull-requests.service.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export class VssPullRequests {
         };
         return Promise.all(repos.map(repo => {
             return this.client.getPullRequests(repo.id, search).then(prs => {
-                return prs.map(pr => {
+                return prs.filter(pr => !pr.isDraft).map(pr => {
```
Comment (joelbourbon) [inline src/vss-pull-requests.service.ts]: Needs to be tested before this PR gets merged.

Comment (joelbourbon) [reply src/vss-pull-requests.service.ts]: Just tested it on a personal Azure Devops by publishing it privately to my own Publisher.

**Comment by joelbourbon** [comment]
> I would like to

Done!

File: changelog.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Version 1.0.9
+- Added this changelog.
+- Now filtering out the PRs that are in Draft mode.
```
Comment (nshenoy) [inline changelog.md]: Nice! Thanks for adding the changelog.

File: mimeo-vsts-extensions.json [hunk 3]
```diff
@@ -87,10 +87,9 @@
       "addressable": true,
       "packagePath": "lib"
     }
- 
   ],
   "categories": [
-    "Code"
+    "Azure Repos"
```
Comment (nshenoy) [inline mimeo-vsts-extensions.json]: Thanks for this.

File: src/vss-pull-requests.service.ts (hunk 1 above)
Comment (nshenoy) [reply src/vss-pull-requests.service.ts]: I appreciate that you tested this out! Thank you so much! Makes it much easier.

**Comment by nshenoy** [comment]
Approved! This has been merged. I'll try to publish this shortly.

**Comment by joelbourbon** [comment]
Thanks for the quick publish @nshenoy !!! We needed that!

**Pull request merged**